homeassistant/components/device_tracker/google_maps.py

```diff
@@ -38,6 +38,9 @@
     vol.Required(CONF_PASSWORD): cv.string,
     vol.Required(CONF_USERNAME): cv.string,
     vol.Optional(CONF_MAX_GPS_ACCURACY, default=100000): vol.Coerce(float),
+    vol.Optional(CONF_DEVICES, default={}): vol.Any(None, vol.Schema({
```

Comment:
It should already be possible to change the name via known_devices.yaml. Isn't that sufficient?

Reply:
Names are great for the UI, but automations use IDs.

Reply:
We should add a unique ID so users can change entity_id in the UI.

Reply:
Yeah I thought about that too @balloob. I just assumed there's something special with known devices that made them different from other entities. Am I wrong?

Reply:
This is an architecture comment, and not really on topic, but I would like us to remake the device tracker component so it is line with the other entity components, so that developers have the entity interface accessible. It will mean we could set a unique_id etc.

Reply:
I 100% agree.